quinn-proto/src/config.rs

```diff
@@ -824,6 +824,8 @@ pub struct ServerConfig {
     pub(crate) incoming_buffer_size_total: u64,
 }
 
+const DEFAULT_RETRY_TOKEN_LIFETIME_SECS: u64 = 15;
```

Comment:
Why extract this? There is no motivation in this commit. (And at least for this commit it could just be a `Duration`.)

(If we're going to do this, let's dump the `const` to the bottom of the module.)

Reply:
Reverted. It doesn't matter much I think, so I'll just be consistent with the existing code.